exec.py

```diff
@@ -48,11 +48,12 @@ def init_submodule_if_not(options):
                 print(f'Failed to update submodule "{sub}"')
 
 
-def run_kartotherian(options):
+def run_kartotherian(docker_base_params, options):
```

Comment:
hum this change should not be here, or am I missing something ?

Reply:
No, I may have been early with making this PR reviewable.

I just finished testing it more exhaustively (running several kinds of commands with several "namespaces")